<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>produitang</class>
 <widget class="QDialog" name="produitang">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1827</width>
    <height>753</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>261</width>
     <height>941</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(163, 163, 163);</string>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/Downloads/home (1).svg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 30px;

color: black;
padding: 0 8px;
background-color: rgb(163, 163, 163);
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
    </property>
    <property name="text">
     <string>HOME</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/Downloads/align-justify.svg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>250</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 30px;

color: black;
padding: 0 8px;
background-color: rgb(163, 163, 163);
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
    </property>
    <property name="text">
     <string>TACHE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>330</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 30px;

color: black;
padding: 0 8px;
background-color: rgb(163, 163, 163);
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
    </property>
    <property name="text">
     <string>SETTINGS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/Downloads/settings.svg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 20px;
color: black;
padding: 0 8px;
background: #ebc5c5;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
    </property>
    <property name="text">
     <string>MEDMOVE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_24">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>920</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 30px;
background-color: rgb(255, 69, 72);

color: black;
padding: 0 8px;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
    </property>
    <property name="text">
     <string>Fran√ßais</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_25">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>920</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 30px;

color: black;
padding: 0 8px;
font: bold, &quot;Comic Sans MS&quot;;;
background-color: rgb(188, 24, 27);
text-align:center;</string>
    </property>
    <property name="text">
     <string>English</string>
    </property>
   </widget>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>50</y>
     <width>1461</width>
     <height>731</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans</family>
     <pointsize>11</pointsize>
     <italic>false</italic>
     <stylestrategy>PreferDefault</stylestrategy>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QTabBar::tab {
	border-color: rgb(170, 170, 0);
	background-image: url(:/Downloads/back.png);
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #E1E1E1, stop: 0.4 #DDDDDD,
                                stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);
    border: 2px solid#C4C4C3;
    border-bottom-color: #C2C7CB; /* same as the pane color */
    border-top-left-radius: 4px;
    border-top-right-radius: 4px;
    min-width:20ex;
    padding: 2px;
     font: bold, &quot;Comic Sans MS&quot;;
}

QTabBar::tab:selected, QTabBar::tab:hover {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #8f8c8d, stop: 0.4 #8f8c8d,
                                stop: 0.5 #8f8c8d, stop: 1.0 #8f8c8d);
}

QTabBar::tab:selected {
    border-color: #9B9B9B;
    border-bottom-color: #C2C7CB; /* same as pane color */
}</string>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Triangular</enum>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>add</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>110</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 20px;
color: black;
padding: 0 8px;
background: #ebc5c5;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>290</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 20px;
color: black;
padding: 0 8px;
background: #ebc5c5;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
     </property>
     <property name="text">
      <string>state</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_11">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>230</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 20px;
color: black;
padding: 0 8px;
background: #ebc5c5;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
     </property>
     <property name="text">
      <string>availability</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_12">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>170</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 20px;
color: black;
padding: 0 8px;
background: #ebc5c5;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
     </property>
     <property name="text">
      <string>Brand</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nom">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>110</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid black;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
padding: 0 8px;
background: white;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="marque">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>170</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid black;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
padding: 0 8px;
background: white;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="disponibilite">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>230</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid black;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
padding: 0 8px;
background: white;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="etat">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>290</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid black;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
padding: 0 8px;
background: white;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="ajout">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>360</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 20px;
color: black;
padding: 0 8px;
background: #E1E1E1;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
     </property>
     <property name="text">
      <string>SAVE</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_14">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>60</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 20px;
color: black;
padding: 0 8px;
background: #ebc5c5;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
     </property>
     <property name="text">
      <string>Id</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="id">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>60</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid black;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
padding: 0 8px;
background: white;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>view</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_15">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/Downloads/search (1).svg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="nomrecherche">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>30</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid black;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
padding: 0 8px;
background: white;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_16">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/Downloads/arrow-down.svg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_17">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>140</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/Downloads/file-text (1).svg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_26">
     <property name="geometry">
      <rect>
       <x>980</x>
       <y>420</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 20px;
color: black;
padding: 0 8px;
background: #E1E1E1;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
     </property>
     <property name="text">
      <string>QRcode</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_19">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>210</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/Downloads/bar-chart-2.svg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QTableView" name="view">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>641</width>
       <height>441</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nom1">
     <property name="geometry">
      <rect>
       <x>1180</x>
       <y>100</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid black;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
padding: 0 8px;
background: white;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="etat1">
     <property name="geometry">
      <rect>
       <x>1180</x>
       <y>280</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid black;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
padding: 0 8px;
background: white;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="disponibilite1">
     <property name="geometry">
      <rect>
       <x>1180</x>
       <y>220</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid black;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
padding: 0 8px;
background: white;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="marque1">
     <property name="geometry">
      <rect>
       <x>1180</x>
       <y>160</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid black;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
padding: 0 8px;
background: white;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_34">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/Downloads/arrow-up.svg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="ajout_4">
     <property name="geometry">
      <rect>
       <x>1250</x>
       <y>320</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/Downloads/save (1).svg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="ajout_3">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>620</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/Downloads/trash-2 (1).svg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_29">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>631</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 20px;
color: black;
padding: 0 8px;
background: #ebc5c5;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
     </property>
     <property name="text">
      <string>Id</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="id1">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>630</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid black;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
padding: 0 8px;
background: white;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="id1_3">
     <property name="geometry">
      <rect>
       <x>1180</x>
       <y>40</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid black;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
padding: 0 8px;
background: white;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_13">
     <property name="geometry">
      <rect>
       <x>1140</x>
       <y>480</y>
       <width>111</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/Downloads/2222.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label1">
     <property name="geometry">
      <rect>
       <x>960</x>
       <y>450</y>
       <width>171</width>
       <height>151</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_18">
     <property name="geometry">
      <rect>
       <x>1070</x>
       <y>220</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 20px;
color: black;
padding: 0 8px;
background: #ebc5c5;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
     </property>
     <property name="text">
      <string>availability</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_20">
     <property name="geometry">
      <rect>
       <x>1070</x>
       <y>280</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 20px;
color: black;
padding: 0 8px;
background: #ebc5c5;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
     </property>
     <property name="text">
      <string>state</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_21">
     <property name="geometry">
      <rect>
       <x>1070</x>
       <y>160</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 20px;
color: black;
padding: 0 8px;
background: #ebc5c5;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
     </property>
     <property name="text">
      <string>Brand</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_22">
     <property name="geometry">
      <rect>
       <x>1070</x>
       <y>100</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 20px;
color: black;
padding: 0 8px;
background: #ebc5c5;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_23">
     <property name="geometry">
      <rect>
       <x>1070</x>
       <y>40</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 20px;
color: black;
padding: 0 8px;
background: #ebc5c5;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
     </property>
     <property name="text">
      <string>Id</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_27">
     <property name="geometry">
      <rect>
       <x>1260</x>
       <y>530</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/Downloads/edit.svg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
